examples/getstarted/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "getstarted",
   "private": true,
-  "version": "3.2.2",
+  "version": "3.2.3",
```

Comment:
Other commits

Reply:
Hey Derrick, sorry I lost the thing here. What should I do exactly to merge this?